src/OpenLoco/src/Objects/ObjectManager.h

```diff
@@ -197,6 +200,8 @@ namespace OpenLoco::ObjectManager
     void updateTerraformObjects();
     void updateLastTrackTypeOption();
 
+    const std::array<LandObjectFlags, getMaxObjects(ObjectType::land)>& getLandObjectFlagsCache();
```

Comment:
Can we come up with a type alias for this? Even just `LandObjectFlagsCollection` or some such.